### ISSUE TITLE
Add some statistical charts

### ISSUE THREAD

**Issue by LondonClass** [MyreMylar/pygame_gui#517]
**Is your feature request related to a problem? Please describe.**
Sometimes we will display some statistical information in the game.

**Describe the solution you'd like**
Add pie charts, bar charts, line charts, and other similar items.

**Comment by MyreMylar** [comment]
I agree these would be cool!

I also want to support nice tables of data (I'm a spreadsheet nerd).

**Comment by GimLala** [comment]
I agree with that too, although I'm interested how you'd implement tables with good performance because tables tend to become large very quickly (especially when loaded from databases) which means the number of elements would grow rapidly along with it.
